.circleci/config.yml

```diff
@@ -535,6 +535,7 @@ jobs:
             export CUDA_HOME=/usr/local/cuda
             mrp logs &
             mrp up grasp_server segmentation_server --nobuild
+            sleep 5
```

Comment:
A little worried that this sleep might cause tests to fail in the future if the instance happens to run slow. Maybe replace with a script that pings the servers until timeout?

Reply:
@lshamis Is there a simple way to query mrp/a0 RPC server whether it's alive / ready to accept calls?

Reply:
Added this